builder/virtualbox/ovf/builder.go

```diff
@@ -48,6 +50,9 @@ func (b *Builder) Run(ui packer.Ui, hook packer.Hook, cache packer.Cache) (packe
 	state.Put("hook", hook)
 	state.Put("ui", ui)
 
+	// Get extension from SourcePath
+	extension := filepath.Ext(b.config.SourcePath)
```

Comment:
Secretly, I want this builder to support box files, but I've demonstrated this not at all. I'll set it to "ova".